cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICartoDBSink.java

```diff
@@ -618,18 +619,18 @@ protected String buildTableName(String servicePath, String entity, String attrib
                             + "dm-by-service-path data model");
                 } // if
 
-                name = NGSIUtils.encodePostgreSQL(servicePath, true);
+                name = NGSICharsets.cartoDBEncode(servicePath.substring(1));
```

Comment:
A very important thing regarding this, and disccussed offline: encoding the slash as `x002f` allows this sink to work with a `data_model=dm-by-service-path` and root subservice combination.

Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1057/commits/b96d27be65902c5dd7e824d6f6f5a0ca099514bd